### ISSUE TITLE
Split XLib.jl and XClipboard.jl

### ISSUE THREAD

**Issue by daringer** [kmsquire/XClipboard.jl#4]
Was looking for an XLib API, found only yours a little hidden behind the XClipboard package. 
Having them separated would make sense imho, as XLib does not necessarily need a clipboard...
What do you think? registering a new pkg is just a pull-request away, right?

So if you're ok with it I could register XLib.jl and maintain XLib.jl, so your XClipboard.jl just needs a "using" instead of an include() ? Feels like stealing code ;) You could also do this by yourself, just an offer...

**Comment by kmsquire** [comment]
Thanks!  I think this is a great idea, and if you're open to it, I'd rather just transfer ownership of XClipboard.jl to you--I haven't used it much (and was disappointed at the limited functionality available). 

Just a warning, I also found the xlib stuff a little challenging to work with. If I recall, most of the public docs assume a 32 bit system, and the data structure sizes change on 64 bit. I got to a working library through a bit of trial and error.
